### PR TITLE
Auto Deploy Reliability

### PR DESCRIPTION
**Pull request by bengl**
Automatically deploy `master` and `testing` to reliability env.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -161,6 +161,15 @@ prebuild-job: &prebuild-job
         - master
         - /v\d+\.\d+/
 
+deply-reliability:
```
Comment (brettlangdon) [inline .circleci/config.yml:164]:
```suggestion
deploy-reliability:
```

Comment (rochdev) [reply .circleci/config.yml:164]: deeply reliable

Comment (bengl) [reply .circleci/config.yml]: fixed in latest push

Comment (rochdev) [reply .circleci/config.yml]: Looks like the typo was in multiple places.

Comment (bengl) [reply .circleci/config.yml]: Does it? I can't find `deply` anywhere else.

**Pull request merged**